jbi/jira/service.py

```diff
@@ -118,14 +118,22 @@ def add_jira_comment(self, context: ActionContext):
         """Publish a comment on the specified Jira issue"""
         context = context.update(operation=Operation.COMMENT)
         commenter = context.event.user.login if context.event.user else "unknown"
-        comment = context.bug.comment
-        assert comment  # See jbi.steps.create_comment()
-        assert comment.body  # Also see jbi.steps.create_comment()
+
+        if context.event.target == "attachment":
+            routing_key = (
+                context.event.routing_key or ".modified"
+            )  # only for type checking.
+            _, verb = routing_key.rsplit(".", 1)
+            formatted_comment = f"*{commenter}* {verb}d an attachment"
```

Comment:
If I'm reading this correctly, when a user modifies an attachment this will say something like:
Jane Jetson modifie*dd* an attachment

Maybe a little string replace line to clean up the extra "d" is all we need?

Reply:
It will be `modifyed` which is not great either